colabfold/batch.py

```diff
@@ -1,5 +1,7 @@
 import os
 
+from Bio.PDB import MMCIFParser
```

Comment:
Do we have an explicit dependency in poetry on biopython?

Reply:
yes.